tests/core/data/test_sql_data_node.py

```diff
@@ -108,10 +109,10 @@ def test_read(self, mock_read_as, mock_read_as_pandas_dataframe, mock_read_as_nu
                 "db_engine": "mssql",
                 "read_query": "SELECT * from table_name",
```

Comment:
The comment on line 101 is not up to date anymore. Please update it, and maybe you can create a case to test with the custom expose_type